cmd/minishift/cmd/openshift/config.go

```diff
@@ -27,8 +27,8 @@ const (
 
 var configCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Displays or patches OpenShift configuration.",
-	Long:  "Displays or patches OpenShift master or node configuration. Patches are supplied in JSON format.",
+	Short: "Displays or updates the OpenShift configuration.",
```

Comment:
I don't think this is correct and we had the same discussion for the docs. In this case we are talking about patching which means more than updating. I cannot recall where we agreed that patching == updating